De-json-apis-and-ajax.json

```diff
@@ -0,0 +1,542 @@
+			"  </div>",
+			"  <div class = \"row text-center\">",
+			"    <div class = \"col-xs-12 well message\">",
+			"      The message will go here",
+			"    </div>",
+			"  </div>",
+			"  <div class = \"row text-center\">",
+			"    <div class = \"col-xs-12\">",
+			"      <button id = \"getMessage\" class = \"btn btn-primary\">",
+			"        Get Message",
+			"      </button>",
+			"    </div>",
+			"  </div>",
+			"</div>"
+		],
+		"tests": [
+			"assert(code.match(/\\$\\s*?\\(\\s*?(?:'|\")\\#getMessage(?:'|\")\\s*?\\)\\s*?\\.on\\s*?\\(\\s*?(?:'|\")click(?:'|\")\\s*?\\,\\s*?function\\s*?\\(\\s*?\\)\\s*?\\{/gi), 'message: Bind the click event to the button with the ID of <code>getMessage</code>');",
+			"assert(code.match(/\\n*?\\s*?\\}\\n*?\\s*?\\);/gi) && code.match(/\\n*?\\s*?\\}\\);/gi).length >= 2, 'message: Be sure to close your functions with <code>}&#41;;</code>');"
+		],
+		"type": "waypoint",
+		"challengeType": 0,
+		"titleEs": "Activa eventos de pulsaci�n con jQuery",
+		"descriptionEs": [
+			"En esta secci�n, vamos a aprender c�mo obtener datos de las APIs. Las APIs - o interfaces de programaci�n de aplicaciones - son herramientas que utilizan los computadores para comunicarse entre s�.",
+			"Tambi�n aprenderemos c�mo actualizar HTML con los datos que obtenemos de estas API usando una tecnolog�a llamada Ajax.",
+			"En primer lugar, vamos a revisar lo que hace la funci�n <code>$(document).ready()</code>. Esta funci�n hace que todo el c�digo dentro de ella se ejecute s�lo hasta que nuestra p�gina ha sido cargada.",
+			"Hagamos que nuestro bot�n \"Get message\" cambie el texto del elemento con clase <code>message</code>.",
+			"Antes de poder hacer esto, tenemos que implementar un <code>evento de pulsaci�n</code> dentro de nuestra funci�n <code>$(document).ready()</code>, a�adiendo este c�digo:",
+			"<blockquote>$(\"#getMessage\").on(\"click\", function(){</br></br>});</blockquote>"
+		],
+		"titlePt": "Ativando eventos de clique com jQuery",
+		"descriptionPt": [
+			"Nesta sess�o, vamos aprender como obter dados de uma API. As APIS - Interface de Programa��o de Aplicativos - s�o ferramentas usadas pelos computadores  para se comunicarem entre si.",
+			"Tamb�m aprenderemos como utilizar o HTML com os dados obtidos de uma API usando uma tecnologia chamada Ajax",
+			"Em primeiro lugar, vamos revir o que faz a fun��o <code>$(document).ready()</code>. Esta fun��o faz com que todo o codigo que esteja dentro de seu escopo execute somente quando a nossa p�gina tenha sido carregada.",
+			"Vamos fazer nosso but�o \"Get message\" mudar o texto do elemento com a classe <code>message</code>.",
+			"Antes de poder fazer isso, temos que implementar um <code>evento de clique</code> dentro da nossa fun��o <code>$(document).ready()</code>, adicionando este c�digo:",
+			"<blockquote>$(\"#getMessage\").on(\"click\", function(){</br></br>});</blockquote>"
+		],
+		"titleDe": "Klickereignisse mit jQuery ausl�sen",
```

Comment:
Bin mir nicht sicher, ob "Klickereignisse" ein wenig zu sehr "eingedeutscht" klingt. Hätte "Klick-Events" bevorzugt, da "Events" auch in Deutsch ein gängiger Begriff ist und insgesamt der Begriff "Events" kohärent mit anderen Übersetzungen verwendet wird.